wasp-app-runner/tsconfig.json

```diff
@@ -12,7 +12,9 @@
     "noUncheckedIndexedAccess": true,
     "moduleResolution": "NodeNext",
     "module": "NodeNext",
+    "rootDir": "src",
     "outDir": "dist",
+    "declaration": true,
     "sourceMap": true,
```

Comment:
Seems like we don't need these two. You can comment them out and explain why if you want.